test/extended/authentication.sh

```diff
@@ -59,34 +59,132 @@ oc new-project openldap
 # create all the resources we need
 oc create -f test/extended/fixtures/ldap
 
+is_event_template=(                      \
+"{{with \$tags := .status.tags}}"        \
+	"{{with \$event := index \$tags 0}}" \
+		"{{\$event.tag}}"                \
+	"{{end}}"                            \
+"{{end}}"                                \
+)
+is_event_template=$(IFS=""; echo "${is_event_template[*]}") # re-formats template for use
+
 # wait until the last event that occured on the imagestream was the successful pull of the latest image
-wait_for_command 'oc get imagestream openldap --template="{{with \$tags := .status.tags}}{{with \$event := index \$tags 0}}{{\$event.tag}}{{end}}{{end}}" | grep latest' $((60*TIME_SEC))
+wait_for_command 'oc get imagestream openldap --template="${is_event_template}" | grep latest' $((60*TIME_SEC))
 
 # kick off a build and wait for it to finish
 oc start-build openldap --follow
 
-# wait for the deployment to be up and running
-wait_for_command 'oc get pods -l deploymentconfig=openldap-server --template="{{with \$items := .items}}{{with \$item := index \$items 0}}{{\$item.status.phase}}{{end}}{{end}}" | grep Running' $((60*TIME_SEC))
+server_ready_template=(                                       \
```

Comment:
Hopefully this is easier to read while we find a cleaner (?) way of doing this. @deads2k